### DISCUSSION TITLE
Surface Lacquer?

### DISCUSSION THREAD

**Discussion by Emma-Walker** [BEEmod/BEE2-items#4021] [General]
I have 2 questions regarding Surface Lacquer.

1. How many entities does Surface Lacquer use?
2. Is it possible to make it non-shiny? I would prefer normal looking walls that simply can't be painted on. I understand the logic behind shinyness, however I only want to prevent accidental painting on side walls.

**Comment by vrad-exe** [comment]
It doesn't use any entities, it just swaps the surface to an alternate material which is set as unpaintable. The shiny texture is there to make sure players can tell that a surface is unpaintable, not having that would just be confusing. Imagine if white surfaces were randomly non-portalable with no indication of that until you tried to place a portal on it.

**Comment by Emma-Walker** [comment]
I understand your reasoning. However, in the case of gel, it's different. Gel is notoriously misbehaving and leaking and splashing into places where it shouldn't be. Especially white gel can then be used to shoot portals where they lead to a wrong solution. You suggest to paint every possible spot around tractor beams and flip panels in a shiny paint to prevent drops from flying to these places. I think that shinyness is more confusing to the player than noticing that gel doesn't stick where it shouldn't be sticking anyway.

I love the idea of Surface Lacquer. In fact, I am the one who posted https://github.com/BEEmod/BEE2-items/issues/3301

But the shinyness makes it unusable for me. Please let chamber creators decide whether they want it shiny or not.

>Imagine if white surfaces were randomly non-portalable

I don't understand. They wouldn't be white in the first place. They would remain dark. Just the shine were gone. There would be simply no gel sticking on them.

**Comment by TeamSpen210** [comment]
The problem is that there wouldn't then be a way to tell if the surface can't be painted simply by observing it, only by actually painting it. If I'm trying to solve the puzzle, that would let me know that I shouldn't bother trying to paint that surface, instead of thinking I can, putting work into doing so, then getting confused/disappointed when it doesn't work. The portalability example was a comparison. Imagine if Portal didn't have black/white surfaces at all, and instead the only way to know would be to shoot and see if it fails.

**Comment by Emma-Walker** [comment]
> The problem is that there wouldn't then be a way to tell if the surface can't be painted simply by observing it,

I think that should still be the decision of the chamber creator. I don't think any chamber creator will use Surface Lacquer for any other purpose than prevention of splashes around tractor beams and so on. 

I think the danger of confusion is bigger right now. If a player sees a shiny surface, they will wonder what's the purpose of it. And if a player accidentally splashes white gel in a place where it shouldn't be, they might also wonder whether that was the intended solution.

**Comment by slambo312** [comment]
Why not just make it so the lacquer is orange if it's on a portalable surface, and blue if it's not (similar to antlines, but covering full panels, or some crap).

**Reply by vrad-exe** [reply]
How would that help anything?

**Reply by slambo312** [reply]
it'd make the lacquer more visible, and not be all shiny.
